src/pages/DatasetCatalog.js

```diff
@@ -392,23 +393,23 @@ class DatasetCatalog extends Component {
               table({ className: 'table' }, [
                 thead({}, [
                   tr({}, [
+                    th({ isRendered: this.state.isAdmin, className: 'cell-size', style: { textAlign: 'center' } }, ['Actions']),
                     th({}),
-                    th({ isRendered: this.state.isAdmin, className: 'table-titles dataset-color cell-size', style: { textAlign: 'center' } }, ['Actions']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Dataset ID']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Dataset Name']),
-                    th({ className: 'table-titles dataset-color cell-size'}, ['Data Access Committee']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Data Source']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Structured Data Use Limitations']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Data Type']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Disease Studied']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Principal Investigator (PI)']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['# of Participants']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Description']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Species']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Data Custodian']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Consent ID']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['SC-ID']),
-                    th({ isRendered: this.state.isAdmin, className: 'table-titles dataset-color cell-size' }, ['Approved Requestors'])
+                    th({ className: 'cell-size' }, ['Dataset ID']),
+                    th({ className: 'cell-size' }, ['Dataset Name']),
+                    th({ className: 'cell-size'}, ['Data Access Committee']),
```

Comment:
super minor spacing nit: 
```suggestion
                    th({ className: 'cell-size' }, ['Data Access Committee']),
```